### PR TITLE
Feat - my tickets page updates and other fixes.

### PR DESCRIPTION
**Pull request by rafsuntaskin**
### 🎫 Ticket

[TICKET_ID] 

### 🗒️ Description

* Spreadsheet issues fixed: 81, 112, 118, 61, 123, 34, 31, 120, 30.

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

### CODE REVIEW

File: src/Tickets/Flexible_Tickets/template-tags.php [hunk 1]
```diff
@@ -165,4 +165,58 @@ function tec_tickets_get_default_ticket_type_label_lowercase( string $context =
...
+	/**
+	 * Returns the filtered default Ticket Type label in plural.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $context The context in which this string is filtered, e.g. 'verb' or 'template.php'.
+	 *
+	 * @return string The plural version of default ticket type label.
+	 */
+	function tec_tickets_get_default_ticket_type_label_plural( string $context = '' ): string {
```
Comment (lucatume) [inline src/Tickets/Flexible_Tickets/template-tags.php:180]: Question: should this be here or in the more general ET template tags? Is this exclusive of the FT feature?

File: src/Tickets/Flexible_Tickets/template-tags.php [hunk 2]
```diff
@@ -165,4 +165,58 @@ function tec_tickets_get_default_ticket_type_label_lowercase( string $context =
...
+	/**
+	 * Returns the filtered default Ticket Type label in plural and lowercase.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $context The context in which this string is filtered, e.g. 'verb' or 'template.php'.
+	 *
+	 * @return string The plural and lowercase version of default ticket type label.
+	 */
+	function tec_tickets_get_default_ticket_type_label_plural_lowercase( string $context = '' ): string {
```
Comment (lucatume) [inline src/Tickets/Flexible_Tickets/template-tags.php:207]: Same question for this template tag: should this be here or in the more general ET template tags?

File: src/Tickets/Flexible_Tickets/template-tags.php (hunk 1 above)
Comment (rafsuntaskin) [reply src/Tickets/Flexible_Tickets/template-tags.php]: Good call, [moved](https://github.com/the-events-calendar/event-tickets/pull/2875/commits/f27d4f7224a461dd472563f05eb5383bcfaeaa27) them to base template tags for tickets:

File: src/Tickets/Flexible_Tickets/template-tags.php (hunk 2 above)
Comment (rafsuntaskin) [reply src/Tickets/Flexible_Tickets/template-tags.php]: Good call, [moved](https://github.com/the-events-calendar/event-tickets/pull/2875/commits/f27d4f7224a461dd472563f05eb5383bcfaeaa27) them to base template tags for tickets:

**Review by lucatume — APPROVED** [review]

**Pull request merged**